.travis.yml

```diff
@@ -30,13 +31,19 @@ before_script:
   # point to fold-grammars source directory:
   - sed -i "s#rootDir = '/vol/fold-grammars/src/'#rootDir = '`pwd`/'#" Misc/Applications/lib/foldGrammars/Settings.pm
 script:
-  - if [[ "$TEST_SUBTASK" == "default" ]]; then
+  - if [[ "$TEST_SUBTASK" == "unambig" ]]; then
     pushd .;
-    cd Misc/Test-Suite/GeorgStyle/;
-    bash run.sh ../Truth;
+    cd Misc/Test-Suite/;  
+    python runTests.py;
```

Comment:
it is strange to see in the log, that the compilation process errored, but Travis reports the test as passing?!
Maybe, because it only checks the exit code of the last command? If so, `python runTests.py` fails, but the `popd` command is successful and results in Travis wrongly assuming everything is good. Can you check what happens if you replace `;` with `&&` ? It might be wrong in my code block as well!